packages/p2p/src/components/buy-sell/buy-sell-form.jsx

```diff
@@ -33,19 +34,16 @@ BuySellFormReceiveAmount.propTypes = {
 const BuySellForm = observer(props => {
     const isMounted = useIsMounted();
     const { buy_sell_store } = useStores();
-    const { setPageFooterParent } = props;
-
     buy_sell_store.setFormProps(props);
 
-    const {
-        advertiser_details,
-        description,
-        local_currency,
-        max_order_amount_limit_display,
-        min_order_amount_limit,
-        min_order_amount_limit_display,
-        price,
-    } = buy_sell_store?.advert;
+    const [limits, setLimits] = React.useState({
+        max_order_amount_limit_display: buy_sell_store?.advert?.max_order_amount_limit_display,
+        min_order_amount_limit: buy_sell_store?.advert?.min_order_amount_limit,
+        min_order_amount_limit_display: buy_sell_store?.advert?.min_order_amount_limit_display,
+    });
+
+    const { setPageFooterParent } = props;
+    const { advertiser_details, description, local_currency, price } = buy_sell_store?.advert;
```

Comment:
Do we really need optional chaining here?

Reply:
We do, but I notice another error. We should have a fallback value else we'll get a `Cannot destructure property "x, y, z" as it is undefined.` Let me add this.

```suggestion
    const { advertiser_details, description, local_currency, price } = buy_sell_store?.advert || {};
```